src/components/LocalizationMenu/LocalizationMenu.jsx

```diff
@@ -28,8 +30,8 @@ export default class LocalizationMenu extends React.Component {
     return (
       <div style={{ minWidth: '200px' }}>
         <LanguageContainerDiv>
-          {Object.entries(SUPPORTED_LANGUAGES).map(([name, lang]) => (
-              <a href={`/${lang}/`}>
+          {Object.entries(this.formatLanguages).map(([name, lang]) => (
+              <a key={name} href={`/${lang}/`}>
```

Comment:
This line is being edited because React needs for each anchor to have a unique key.